### PR TITLE
Resolves some memory display fixes in health monitor.

### PR DESCRIPTION
**Pull request by pveentjer**
availableMemory is not used (also not in previous releases) so removed the field from the health monitor.

Some fields in the healthmonitor.metrics were not updated correctly.

### CODE REVIEW

**Comment by pveentjer** [comment]
run-lab-run

File: hazelcast/src/main/java/com/hazelcast/internal/diagnostics/HealthMonitor.java [hunk 1]
```diff
@@ -274,8 +274,8 @@ private void logDiagnosticsHint() {
         private double memoryUsedOfMaxPercentage;
 
         public void update() {
-            memoryUsedOfTotalPercentage = PERCENTAGE_MULTIPLIER * runtimeUsedMemory.read() / runtimeTotalMemory.read();
-            memoryUsedOfMaxPercentage = PERCENTAGE_MULTIPLIER * runtimeUsedMemory.read() / runtimeMaxMemory.read();
+            memoryUsedOfTotalPercentage = (PERCENTAGE_MULTIPLIER * runtimeUsedMemory.read()) / runtimeTotalMemory.read();
```
Comment (jerrinot) [inline hazelcast/src/main/java/com/hazelcast/internal/diagnostics/HealthMonitor.java]: it's not not related to this very change, but can't `runtimeTotalMemory` ever return 0?

**Comment by jerrinot** [comment]
:+1:

**Comment by pveentjer** [comment]
run-lab-run

**Comment by pveentjer** [comment]
run-lab-run

**Comment by pveentjer** [comment]
run-lab-run

**Comment by pveentjer** [comment]
run-lab-run

**Pull request merged**